user-guide/Advanced_Modules/User Defined APIs/Defining_an_API/UD_APIs_Define_New_API.md

```diff
@@ -151,25 +151,30 @@ You can also use the `StatusCode` enum, which contains suggestions, and cast tha
 > - A simple example is included in the [Automation script example](#creating-the-api-automation-script).
 > - For more insight into which HTTP status codes to use in which circumstances, see [HTTP status codes](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status).
 
-## Creating an API definition and token(s)
+## Creating an API and token(s) in Automation
 
-1. Open your API script in the Automation app in DataMiner Cube and click *Configure API* at the bottom of the window.
+1. Open your API script in the Automation app in DataMiner Cube and click *Configure API...* at the bottom of the window.
 
-   This will open a window where you can create the API definition.
+   This will open a window where you can create the API.
 
-   ![Creating an ApiDefinition](~/user-guide/images/UDPAIS_CreateAPI.jpg)<br>
-   *Creating an API definition in DataMiner 10.3.6*
+   ![Creating an ApiDefinition](~/user-guide/images/UDPAIS_CreateAPI.png)<br>
+   *Creating an API in DataMiner 10.3.6*
+   
+   > [!NOTE]
+   > - The button will be shown in the UI when the the system does support user-defined API (DataMiner 10.3.6.0 onwards) and when the system has an active Indexing Engine.
```

Comment:
... when the system **supports** sounds more natural to me.

Reply:
Also double 'the'

Reply:
Done